src/calculus.jl

```diff
@@ -0,0 +1,10 @@
+
+
+function diff(b1::Basic, b2::Basic)
```

Comment:
I can add a method to get the enum from the class name as a string. `TypeID basic_get_class_id(char *)`. That would be enough right?